scripts/quests/sandoria/DRG_AF2_Chasing_Quotas.lua

```diff
@@ -0,0 +1,272 @@
+local portSandyID = require('scripts/zones/Port_San_dOria/IDs')
+-----------------------------------
+
+local quest = Quest:new(xi.quest.log_id.SANDORIA, xi.quest.id.sandoria.CHASING_QUOTAS)
+
+quest.reward =
+{
+    item     = xi.items. DRACHEN_BRAIS,
+    fame     = 40,
+    fameArea = xi.quest.fame_area.SANDORIA,
+}
+
+quest.sections =
+{
+    -- Section: Quest available.
+    {
+        check = function(player, status, vars)
+            return status == QUEST_AVAILABLE and
+                player:getMainJob() == xi.job.DRG and
+                player:getMainLvl() >= xi.settings.main.AF2_QUEST_LEVEL and
+                player:getQuestStatus(xi.quest.log_id.SANDORIA, xi.quest.id.sandoria.A_CRAFTSMAN_S_WORK) == QUEST_COMPLETED
+        end,
+
+        [xi.zone.PORT_SAN_DORIA] =
+        {
+            ['Arminibit'] =
+            {
+                onTrigger = function(player, npc)
+                    if player:getVar("Quest[0][95]csSeen") == 0 then
+                        return quest:progressEvent(18)
+                    elseif player:getVar("Quest[0][95]csSeen") == 1 then --Declined First Offering.
+                        return quest:progressEvent(14)
+                    end
+                end,
+            },
+
+            ['Ceraulian'] =
+            {
+                onTrigger = function(player, npc)
+                    if player:getVar("Quest[0][95]csSeen") == 0 then
+                        return quest:progressEvent(18)
+                    elseif player:getVar("Quest[0][95]csSeen") == 1 then --Declined First Offering.
+                        return quest:progressEvent(14)
+                    end
+                end,
+            },
+
+            onEventFinish =
+            {
+                [14] = function(player, csid, option, npc)
+                    if option == 1 then
+                        quest:begin(player)
+                        quest:setVar(player, 'csSeen', 0)
+                    else
+                        quest:setVar(player, 'csSeen', 1) -- Declined First Offering.
+                    end
+                end,
+
+                [18] = function(player, csid, option, npc)
+                    if option == 1 then -- Accept quest.
+                        quest:begin(player)
+                        quest:setVar(player, 'csSeen', 0)
+                    else
+                        quest:setVar(player, 'csSeen', 1)
+                    end
+                end,
+            },
+        },
+    },
+
+    -- Section: Quest accepted
+    {
+        check = function(player, status, vars)
+            return status == QUEST_ACCEPTED
+        end,
+
+        [xi.zone.PORT_SAN_DORIA] =
+        {
+            ['Ceraulian'] =
+            {
+                onTrigger = function(player, npc)
+                    local questProgress = quest:getVar(player, 'Prog')
+
+                    if questProgress == 1 then
+                        if player:getCharVar("Timer") > os.time() then
```

Comment:
This variable is set as a quest variable, so it WILL have a quest ID identifier in it's name.

So, basicaly, we are not checking for the correct variable. Would be fixed easily via `quest:getVar(player, 'Timer')`